commons/src/test/java/org/sonarsource/sonarlint/core/commons/log/SonarLintLogTester.java

```diff
@@ -65,6 +65,15 @@
 
   private final Queue<String> logs = new ConcurrentLinkedQueue<>();
   private final Map<Level, Queue<String>> logsByLevel = new ConcurrentHashMap<>();
+  private final boolean writeToStdOut;
+
+  public SonarLintLogTester(boolean writeToStdOut) {
```

Comment:
I don't see this constructor invoked, you want to keep it to ease debugging later on local machine ?

Reply:
Yeah, I used it to investigate a test failure.